src/pages/Filing/UpdateFinancialProfile/types.ts

```diff
@@ -5,6 +5,50 @@ export interface CheckboxOption {
   label: string;
 }
 
+// TODO: Send checkboxes values as {'1': true }?
+// {id: 1, label: "Bank or savings association"},
+// {id: 2, label: "Minority depository institution"},
+// {id: 3, label: "Credit union"},
+// {id: 4, label: "Nondepository institution"},
+// {id: 5, label: "Community development financial institution (CDFI)"},
+// {id: 6, label: "Other nonprofit financial institution"},
+// {id: 7, label: "Farm Credit System institution"},
+// {id: 8, label: "Government lender"},
+// {id: 9, label: "Commercial finance company"},
+// {id: 10, label: "Equipment finance company"},
+// {id: 11, label: "Industrial loan company"},
+// {id: 12, label: "Online lender"},
+// {id: 13, label: "Other"},
+
+export const sblInstitutionTypeMap: Record<string, string> = {
```

Comment:
Agreed, I was experimenting with how we send these enumerated values to the SBL Help team.  Frontend just sending numeric IDs and having those folks need to decode them felt cumbersome for them, since in whatever UI they're using it will probably display the `label` and not the id (just guessing). 
 
The 'short version' (i.e. bankSaving instead of `1`) felt like it may convey enough meaning that they'd be able to take action without needing a lookup table, while keeping it easy for the Frontend to send a short, comma separated list.
 
 But maybe just sending the full label strings (i.e. Bank or savings association) would be best?